client/src/index.js

```diff
@@ -229,14 +249,14 @@ export default function Login() {
                                 />
                             </div>
 
-                            <div className={"usernameTextField"} noValidate id="usernameField">
+                            <div className={"emailTextField"} noValidate id="emailField">
+
                                 <TextField
-                                    id="standard-username"
-                                    label="Username"
+                                    id="email"
+                                    label="Email"
                                     margin="normal"
-                                    autoFocus
                                     fullWidth
-                                    onChange={updateUsername}
+                                    onChange={updatePassword}
```

Comment:
The login doesn't work correctly. Maybe the bug is because you're calling `updatePassword()` instead of `updateEmail()`?